### PR TITLE
ADEN-6587 Remove SourcePoint code from application

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/ARecoveryEngine/ARecoveryEngineApiController.class.php [hunk 1]
```diff
@@ -4,25 +4,13 @@ class ARecoveryEngineApiController extends WikiaController {
...
 
-	public function getDelivery() {
-		$this->loadScript(new ResourceLoaderAdEngineSourcePointCSDelivery());
-	}
+	public function getDelivery() {}
 
-	public function getspmsg() {
-		$this->loadScript(new ResourceLoaderAdEngineSourcePointMessage());
-	}
+	public function getspmsg() {}
```
Comment (kvas-damian) [inline extensions/wikia/ARecoveryEngine/ARecoveryEngineApiController.class.php]: Can we remove those methods?

File: extensions/wikia/ARecoveryEngine/js/adBlockDetection.js [hunk 2]
```diff
@@ -23,10 +23,7 @@ define('ext.wikia.aRecoveryEngine.adBlockDetection', [
 		onBlockingEventsQueue = lazyQueue.makeQueue([], cb),
 		onNotBlockingEventsQueue = lazyQueue.makeQueue([], cb);
 
-	function initEventQueues() {
-		doc.addEventListener('sp.not_blocking', onNotBlockingEventsQueue.start);
-		doc.addEventListener('sp.blocking', onBlockingEventsQueue.start);
-	}
+	function initEventQueues() {}
 
 	function addOnBlockingCallback(callback) {
```
Comment (kvas-damian) [inline extensions/wikia/ARecoveryEngine/js/adBlockDetection.js]: What do you think about removing these callback? I think they're not used now.

File: extensions/wikia/Qualaroo/scripts/Qualaroo.js [hunk 3]
```diff
@@ -98,21 +98,7 @@
...
-		if (!window.ads.runtime || !window.ads.runtime.sp || window.ads.runtime.sp.blocking === undefined) {
-			document.addEventListener('sp.blocking', function () {
-				setAdBlockEnabledAndLoadQualaroo(true);
-			});
-
-			document.addEventListener('sp.not_blocking', function () {
-				setAdBlockEnabledAndLoadQualaroo(false);
-			});
-		} else {
-			setAdBlockEnabledAndLoadQualaroo(window.ads.runtime.sp.blocking);
```
Comment (kvas-damian) [inline extensions/wikia/Qualaroo/scripts/Qualaroo.js:111]: I think we can remove this method as well. I don't believe anyone is using this field in qualaroo

File: extensions/wikia/ARecoveryEngine/js/adBlockDetection.js [hunk 4]
```diff
@@ -23,10 +23,7 @@ define('ext.wikia.aRecoveryEngine.adBlockDetection', [
 		onBlockingEventsQueue = lazyQueue.makeQueue([], cb),
 		onNotBlockingEventsQueue = lazyQueue.makeQueue([], cb);
 
-	function initEventQueues() {
-		doc.addEventListener('sp.not_blocking', onNotBlockingEventsQueue.start);
-		doc.addEventListener('sp.blocking', onBlockingEventsQueue.start);
-	}
+	function initEventQueues() {}
```
Comment (kvas-damian) [inline extensions/wikia/ARecoveryEngine/js/adBlockDetection.js]: we can remove this method

File: extensions/wikia/ARecoveryEngine/ARecoveryEngineApiController.class.php (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/ARecoveryEngine/ARecoveryEngineApiController.class.php]: Yes and done.

File: extensions/wikia/Qualaroo/scripts/Qualaroo.js (hunk 3 above)
Comment (dmnsobczak) [reply extensions/wikia/Qualaroo/scripts/Qualaroo.js:111]: Ok, removed.

File: extensions/wikia/ARecoveryEngine/js/adBlockDetection.js (hunk 4 above)
Comment (dmnsobczak) [reply extensions/wikia/ARecoveryEngine/js/adBlockDetection.js]: True, deleted.

**Comment by kvas-damian** [comment]
Please resolve conflicts

**Comment by dmnsobczak** [comment]
Conflicts resolved.

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**